electron/app/utils.ts

```diff
@@ -100,13 +101,28 @@ export const setProjectsRootFolder = (userHomeDir: string) => {
   }
 };
 
-export const setDeviceID = (deviceID: string) => {
+export const setDeviceID = (deviceID: string, disableTracking: boolean) => {
   const ID: string = electronStore.get('main.deviceID');
 
+  const requestArgs = {
+    method: 'post',
+    body: JSON.stringify({
+      machineId: ID,
+    }),
+    headers: {'Content-Type': 'application/json'},
+  };
+
+  if (!disableTracking) {
+    fetch(`${MONOKLE_INSTALLS_URL}/session`, requestArgs);
+  }
+
   if (!ID) {
     if (NUCLEUS_SH_APP_ID) {
       Nucleus.track(APP_INSTALLED);
     }
+    if (MONOKLE_INSTALLS_URL) {
+      fetch(`${MONOKLE_INSTALLS_URL}/install`, requestArgs);
```

Comment:
So just to understand correctly, this is a workaround for when tracking is disabled which causes `Nucleus.track(APP_INSTALLED);` to not forward the event?